hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/HddsVolume.java

```diff
@@ -146,7 +146,7 @@ public void createWorkingDir(String dirName, MutableVolumeSet dbVolumeSet)
 
     // Create DB store for a newly formatted volume
     if (VersionedDatanodeFeatures.isFinalized(
-        HDDSLayoutFeature.DATANODE_SCHEMA_V3)) {
+        HDDSLayoutFeature.LAST_CHUNK_TABLE)) {
```

Comment:
This change is not required. Let's keep the current code. It's more straight forward.